app/page.js

```diff
@@ -73,7 +73,9 @@ export default function Home() {
             Add Item
           </Typography>
           <Stack direction={"row"} spacing={2}>
-            <TextField id="outlined-basic" label="item" variant="outlined" fullWidth />
+            <TextField id="outlined-basic" label="item" variant="outlined" fullWidth
+              onChange={(e) => setItemName(e.target.value)}
```

Comment:
This is the missing code for doc() without this, adding items to firestore gave me empty path error